tests/prepare/feature/data/check-fips.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -ex
+
+# XXX: How to get the provision method (i.e. the type of the guest)
+# if [[ $PROVISION_METHOD == "container" || $PROVISION_METHOD == "local" ]]; then
+#     echo "Provision method \"$PROVISION_METHOD\" is unsupported."
+#     exit 0
+# fi
+out=$(systemd-detect-virt 2> /dev/null)
+if [[ -z "$out" ]]; then
+    echo "Container is unsupported."
+    exit 0
+fi
+
+if [[ "$STATE" == "enabled" ]]; then
```

Comment:
How about doing this instead? We can't assume English is used as a locale :)
```
--is-enabled: Checks the system FIPS mode status and returns
           failure error code if disabled (2) or inconsistent (1).
```

Reply:
Fips will be covered in a separate PR.